### ISSUE TITLE
Update npm package to support Gen 9

### ISSUE THREAD

**Issue by olanmccarthy** [smogon/damage-calc#503]
Hey guys, just tried downloading the npm package for @smogon/calc and it currently doesn't support generation 9.
It seems to be working fine in this project so was wondering when you are going to release the updated package?

**Comment by BCleveland** [comment]
Same issue here, does anyone know how I could get around this for the time being and have Node use a local version of this repo instead?

**Comment by monsanto** [comment]
The reason the NPM package hasn't been updated is because we don't use the NPM package internally for anything, so over time the process to distribute it decays. Whoever was passionate about maintaining it moved on with life, and the only time we notice is when someone makes a GH issue about it.

I'm not opposed to releasing an NPM package of any of the repos in this organization, but it has to be zero overhead for our organization. So that probably means no meaningful semver (0.x?), and an automated build & upload process as part of GH actions. Or it could be a git branch on here that has the results of a build, which would save us the trouble of maintaining a different set of credentials. If someone wants to work on that then you have my blessing and thanks.

**Comment by JonatasBSM** [comment]
> The reason the NPM package hasn't been updated is because we don't use the NPM package internally for anything, so over time the process to distribute it decays. Whoever was passionate about maintaining it moved on with life, and the only time we notice is when someone makes a GH issue about it.
> 
> I'm not opposed to releasing an NPM package of any of the repos in this organization, but it has to be zero overhead for our organization. So that probably means no meaningful semver (0.x?), and an automated build & upload process as part of GH actions. Or it could be a git branch on here that has the results of a build, which would save us the trouble of maintaining a different set of credentials. If someone wants to work on that then you have my blessing and thanks.


Hey, monsanto! I hope you're doing well. Npm package situation still the same?

**Comment by monsanto** [comment]
@JonatasBSM yes it is

**Comment by JonatasBSM** [comment]
> @JonatasBSM yes it is

I see. Thank for the answer!

**Comment by ajhyndman** [comment]
I did have a look into automating npm releases this weekend.  There is some good documentation with examples using [GitHub Actions](https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages).  My takeaway though is that there is still a bare minimum manual intervention required.  The version number for each module as published must be reflected in the package.json file before publication.

Therefore, someone will need to at least bump the committed version number in the repository to trigger each release.

With that said, the [directions](https://github.com/smogon/damage-calc/blob/master/TASKS.md#releasing-a-new-smogoncalc-package) listed in this repository don't involve much more than that manual version bump, so I'm not sure there's much value to add with automation.

Instead, I have tried my hand at publishing a fork here:

https://www.npmjs.com/package/@ajhyndman/smogon-calc

If anyone wants to update the official publication, that would still be preferred, of course!  The process seems pretty straightforward for someone with access to the right credentials.
